.travis.yml

```diff
@@ -1,11 +1,13 @@
+dist: xenial
```

Comment:
Curious why we want to lock the distribution. Are we jumping ahead or locking to a past distro?

Reply:
This is a jump ahead (default is still Trusty). It's related to failures that we were getting related to switching to `utf8mb4`. MySQL <5.7 (well, actually, just InnoDB) has a problem with indexes on `VARCHAR(255)` (`string`) columns and this character set.

If it were _our_ MySQL server, there's a config option to allow "large indices", but Xenial has MySQL 5.7 so this is the easiest way to sidestep the issue.